.github/workflows/backport.yml

```diff
@@ -10,10 +10,7 @@ jobs:
     runs-on: ubuntu-18.04
     name: Backport
     steps:
-      - name: Backport Bot
-        uses: Gaurav0/backport@v1.0.24
+      - name: Backport
+        uses: zephyrproject-rtos/action-backport@v1.1.99
```

Comment:
nothing to do with this PR, but isn't it bad to make a `.99` release ?
That is a reserved number to indicate that something is not released.
https://github.com/zephyrproject-rtos/action-backport/releases/tag/v1.1.99

Reply:
The reason I use this is that this is a fork of this action:
https://github.com/tibdex/backport

The version number `1.1.99` means "upstream 1.1.0 + our downstream changes" in this context.

Reply:
understood.
although I personally find the use of `1.1.99` a bit misleading.

Also I notice that that all tags: `v1`, `v1.1`, and `v1.1.99` points to the same commit: https://github.com/zephyrproject-rtos/action-backport/commit/d2c343ec11394e07b63d21c70c59fdea9e8ecce3
https://github.com/zephyrproject-rtos/action-backport/tags

![image](https://user-images.githubusercontent.com/9802583/93126506-b776d100-f6cc-11ea-8ea1-497462261a5a.png)

Reply:
>Also I notice that that all tags: v1, v1.1, and v1.1.99 points to the same commi

Yes, this is because of the way the wepack action works. The intent is to keep our releases in lockstep with the upstream ones.

Reply:
got it.
Thanks for explanation.